music21/expressions.py

```diff
@@ -325,6 +325,114 @@ def nextMark(self):
         return RehearsalMark(self.nextContent(), numbering=self.numbering)
 
 
+# ------------------------------------------------------------------------------
+class PedalMark(spanner.Spanner):
+    '''
+        A pedal mark spanner contains a pedaled set of notes.
+        The spanner starts at the moment of down-pedal, and
+        ends at the moment of up-pedal.  There can be any
+        number of pedal "bounces" in the middle of the spanner.
+
+        The visual form of the pedal mark can vary.  The following
+        examples use a pedal mark with one "bounce" in the middle.
+
+        Pedal marks can be lines:            |_______^________|
+        Pedal marks can be normal symbolic:  Ped.    * Ped.   *
+        Pedal marks can be altered symbolic: Ped.    Ped.     *
+        Pedal marks can be symbol and line:  Ped.____^________|
+
+        Pedal marks, whether lines, symbols, or a combination, can
+        represent the sustain pedal (Ped.), the sostenuto pedal
+        (Sost.), the soft (una corda) pedal, or (more rarely) the
+        silent (muting) pedal.
+
+        Pedal marks that are symbolic can be abbreviated: e.g. P. instead
+        of Ped., S. instead of Sost.
+
+        Pedal marks that are lines can have non-printed portions
+        (gaps) in them; these are usually started with "simile",
+        but not necessarily.
+    '''
+    validPedalTypes: tuple[str, ...] = ('sustain', 'sostenuto', 'soft', 'silent')
+    validPedalForms: tuple[str, ...] = ('line', 'symbol', 'altsymbol', 'symline')
```

Comment:
Can we make these be Enums? -- I'm slowing trying to migrate all special strings to be enums.  I'm also thinking maybe enums don't need to be all caps -- it's the one thing I hate about them, how many SCREAMING CAPITALS we're getting in the code.  But in any case, can be do SYMBOL_ALT or SymbolAlt instead of altsymbol and SymbolLine instead of symline -- sym seems like a misspelling of simile line or something.

Great!